flink-end-to-end-tests/test-scripts/test_streaming_sql.sh

```diff
@@ -21,9 +21,6 @@ source "$(dirname "$0")"/common.sh
 
 TEST_PROGRAM_JAR=${END_TO_END_DIR}/flink-stream-sql-test/target/StreamSQLTestProgram.jar
 
-# copy flink-table jar into lib folder
```

Comment:
Why don't wee need this anymore?

Reply:
Copying `flink-table` into the `/lib` was a workaround for a classloading issue that has most likely been fixed a couple of releases ago. @fhueske just to verify it again, we can remove the notice and copying in the end to end test, right?